apps/cow-amm-deployer/src/lib/transactionFactory.ts

```diff
@@ -140,11 +145,25 @@ class CowAmmCreateTx implements ITransaction<creteCowAmmArgs> {
     };
   }
 }
+
+class CowAmmStopTx implements ITransaction<stopCowAmmArgs> {
```

Comment:
any comments on why use `Stop` when the fn name is `Close`? Do you think that's more friendly/meaningful to users?

Reply:
On the UI I thought that the `Stop` would be more friendly, but I don't have a strong opinion here actually